yt_dlp/extractor/streamanity.py

```diff
@@ -0,0 +1,49 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+
+class StreamanityIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?streamanity\.com/video/(?P<id>[A-Za-z0-9]+)'
+    _TEST = {
+        'url': 'https://streamanity.com/video/9DFPTnuYi8f2',
+        'info_dict': {
+            'id': '9DFPTnuYi8f2',
+            'ext': 'mp4',
+            'title': 'Bitcoin vs The Lighting Network',
+            'thumbnail': 'https://res.cloudinary.com/streamanity-next/image/upload/v1631475198/thumb/523908e0-e85b-4555-9050-78a8378835f1.png',
+            'description': '',
+            'uploader': 'Tom Bombadil (Freddy78)',
+        }
+    }
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+#       Download JSON that contains video information
+        video_info = self._download_json('https://app.streamanity.com/api/video/{}'.format(video_id), video_id)
+
+#       All relevant & useful information
```

Comment:
Use try_get for these, especially for non essentials like description thumbnail and title and uploader

Reply:
How exactly to use try_get? Isn't title actually essential?

Reply:
You have to import try_get from utils and then do 
try_get(video_info, lambda x: x['data']['video']['author_name'])

Actually I see that ['data']['video'] is used in all fields so just do 

video_info = self._download_json('https://app.streamanity.com/api/video/{}'.format(video_id), video_id)['data']['video'] as the extraction won't happen if this doesn't exist anyways. And then you can use .get() in fields (no need to import tryget in this case)

Reply:
Title isn't essential as it can be none. Only keys required to get the video are essential, rest can be none.

Reply:
Do I have to use get() in all fields or only non-essential ones? How to use get()? I can't find it in the code

Reply:
You can use get for non essentials. Get is an built-in function for python dicts and you don't have to import it. 
Basically get takes in a key and if it Exists in the dict, then it returns it's value else None or expected value. 

So of a dict is a= { '1':'one'}
Then a.get('1') will return 'one'
a.get('2') will return None
a.get('2', 'two') will return 'two'
And a['2'] will throw error
For more info see 
https://www.w3schools.com/python/ref_dictionary_get.asp

Reply:
I've fixed this